sys/include/net/sock/dtls.h

```diff
@@ -710,6 +706,34 @@ ssize_t sock_dtls_send(sock_dtls_t *sock, sock_dtls_session_t *remote,
  */
 void sock_dtls_close(sock_dtls_t *sock);
 
+/**
+ * @brief Creates a new DTLS session
+ *
+ * Sends a ClientHello to initiate a handshake process with a DTLS server  at @p ep.
```

Comment:
Should we insta-deprecate this?

Reply:
I'm okay with deprecating it right away.